SPECIFICATION.md

```diff
@@ -313,7 +316,8 @@ JSON example:
   "version": 1,
   "data": {
     "name": "Jane Doe",
-    "DoB": 19010101,
+    "dob": 19010101,
+    "phone": "+16170000000",
```

Comment:
You have a plus here (and it looks like plenty of others do this) but https://en.wikipedia.org/wiki/E.164 states 
> As described in by the ITU, the E.164 general format must contain only digits split as follows:
> Country code (1 to 3 digits)
> Subscriber number (max 12 digits)

Reply:
As far as I can tell, the plus is a shorthand for your country's exit code.

Reply:
Strange. I always thought the + was required.